### ISSUE TITLE
Should learning rate be taken into account when continuing training with a previously trained models?

### ISSUE THREAD

**Issue by netw0rkf10w** [AlexeyAB/darknet#650]
Hello,
Thanks for creating a very useful repo, and also for the very active support.
In the README you described how to continue training from a previously trained model (eg. when switching to multiple GPUs). My question is: should one explicitly set a new learning rate according to its value in the last iteration of the previous training?
Thanks a lot!

**Comment by AlexeyAB** [comment]
Hi,

No, you shouldn't change learning rate.

Learning rate will be changed automatically according to the current iteration number that is stored in the weights-file and params in your cfg-file: learning_rate, policy, scales, steps.

**Comment by netw0rkf10w** [comment]
@AlexeyAB That is great! Thank you for your quick reply!
